etl/message/forms.py

```diff
@@ -33,6 +34,10 @@ def clean_html(html: str) -> str:
     soup = BeautifulSoup(html, "html.parser")
     for script in soup(FORBIDDEN_TAGS):
         script.decompose()
+    for img in soup.find_all("img"):
+        img.name = "span"
+        img["class"] = "former_img"
```

Comment:
Maybe we can give it a class name like this "__RATOM__former-img" for sanity and to avoid the very unlikely chance of collision with pre-existing classes?